src/api/config/environments/test.rb

```diff
@@ -20,7 +20,7 @@
 
   # Show full error reports and disable caching
   # local requests don't trigger the global exception handler -> set to false
-  config.consider_all_requests_local = false
+  config.consider_all_requests_local = true
```

Comment:
the comment above that line looks a little misplaced now :)

The rails template just is this:

```
  # Show full error reports and disable caching.
  config.consider_all_requests_local       = true
  config.action_controller.perform_caching = false

  # Raise exceptions instead of rendering exception templates.
  config.action_dispatch.show_exceptions = false

```

And I bet it always has been - but this false has always been in here and noone questioned it so far. There are more puzzling diffs to rails default houghs.

   # Disable request forgery protection in test environment.
-  config.action_controller.allow_forgery_protection = false
+ config.action_controller.allow_forgery_protection = true

(it's interesting that the comment is the same :)

Reply:
Hard to say and probably something worth looking at. But I don't see why this would block the PR.

Regarding the deviation from the default Rails config... It might be similar to [public_file_server.enabled](https://github.com/openSUSE/open-build-service/pull/6616/files). In the Rails  4 to Rails 5 migration guide it was advertised, but the edge Rails guide says something different and is a bit more verbose.

As said, we could review them.

Reply:
blocking the PR is only the misplaced comment saying the opposite to what you do

Reply:
ahh :bulb: